book.common/src/main/java/com/happystays/book/common/events/BookingSuccessEvent.java

```diff
@@ -0,0 +1,29 @@
+package com.happystays.book.common.events;
+
+
+import com.happystays.cqrs.core.dto.CancellationInfo;
+import com.happystays.cqrs.core.dto.PaymentInfo;
+import com.happystays.cqrs.core.dto.UserInfo;
+import com.happystays.cqrs.core.events.BaseEvent;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@SuperBuilder
+public class BookingSuccessEvent extends BaseEvent {
+    private int numberOfRooms;
+    private String ratePlanId;
+    private int guestCount;
+    private String startDate;
+    private String endDate;
+    private PaymentInfo paymentInfo;
+    private String hotelCode;
+    private UserInfo userInfo;
+    private CancellationInfo cancellationInfo;
+    private boolean breakfastIncluded;
+    private String hotelPhone;
```

Comment:
This should contain all the datas that are required to persist in DB, includes the pnr info. Please add all relevant info